versioning/BRM/extentmap.cpp

```diff
@@ -1247,13 +1559,15 @@ void ExtentMap::load(const string& filename, bool fixFL)
 #endif
 
     grabEMEntryTable(WRITE);
+    grabEMIndex(WRITE);
```

Comment:
Won't it be possible to use RAII here  and below so as not to repeat releaseSomething in the right order? Can grabEMIndex throw an exception?

Reply:
Good idea indeed.
JFYI grabEMEntryTable, grabEMIndex, grabFreeList can throw with the same probability so everything must be wrapped up in number of try-catch blocks.